### PR TITLE
Publisher UI improvements.

### PR DESCRIPTION
**Pull request by menakaj**

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/UIComponents/Drawer/Drawer.jsx [hunk 1]
```diff
@@ -28,18 +29,6 @@ class Drawer extends Component {
     constructor() {
         super();
         this.closeDrawer = this.closeDrawer.bind(this);
-        this.scriptId = "drawer";
```
Comment (sinthuja) [inline components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/UIComponents/Drawer/Drawer.jsx:31]: Why is this removed? Though we are moving part of the css classses to the component, there are yet some css that needs to be loaded from the theming (from public) right?

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Create/ApplicationCreate.jsx [hunk 2]
```diff
@@ -218,7 +246,26 @@ class ApplicationCreate extends Component {
...
+                                    <Row>
+                                        <Col>
+                                            <div className="stepper-header-content">
+                                                <div className="step-index">1</div>
+                                                <div className="step-header"> General Info</div>
+                                            </div>
+                                        </Col>
+                                        <Col>
+                                            <span className="step-index">2</span>
+                                            <span className="step-header"> Select Platform</span>
```
Comment (sinthuja) [inline components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Create/ApplicationCreate.jsx]: Please use localization based on react-intl as megala has already integrated it.

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/UIComponents/Drawer/Drawer.jsx (hunk 1 above)
Comment (menakaj) [reply components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/UIComponents/Drawer/Drawer.jsx:31]: Those styles are in the default-theme.css.
drawer.css contains the layout of the drawer content.

Comment (sinthuja) [reply components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/UIComponents/Drawer/Drawer.jsx:31]: ok, then. +1

**Review by sinthuja — APPROVED** [review]
